src/window_main/components/decks/DecksTableComponents.tsx

```diff
@@ -0,0 +1,736 @@
+import _ from "lodash";
+import React, { useRef, useState } from "react";
+import format from "date-fns/format";
+import isValid from "date-fns/isValid";
+import styled from "styled-components";
+import matchSorter from "match-sorter";
+
+import { MANA, CARD_RARITIES } from "../../../shared/constants";
+import { getCardArtCrop, toMMSS, toDDHHMMSS } from "../../../shared/util";
+import RelativeTime from "../../../shared/time-components/RelativeTime";
+import pd from "../../../shared/player-data";
+
+import {
+  formatPercent,
+  formatWinrateInterval,
+  getWinrateClass,
+  getTagColor,
+  showColorpicker
+} from "../../renderer-util";
+import { createInput } from "../../../shared/dom-fns"; // TODO remove this
+import ManaFilter, { ColorFilter } from "../../ManaFilter";
+
+import Aggregator from "../../aggregator";
+import { CSSTransition } from "react-transition-group";
+
+export interface CellProps {
+  cell: any;
+  openDeckCallback: (id: string) => void;
+  archiveDeckCallback: (id: string) => void;
+  tagDeckCallback: (deckid: string, tag: string) => void;
+  editTagCallback: (tag: string, color: string) => void;
+  deleteTagCallback: (deckid: string, tag: string) => void;
+}
+
+export const StyledInputContainer = styled.div.attrs(props => ({
+  className: props.className ? props.className : "" + " input_container"
```

Comment:
```suggestion
className: props.className || " input_container"
```

Reply:
If `" input container"` _has_  to be appended, add some parantheses around the ternary:
```suggestion
className: (props.className ? props.className : "") + " input_container"
```